apps/web/components/NavbarLink.js

```diff
@@ -0,0 +1,66 @@
+// @flow
+
+import * as React from 'react';
+import { View } from 'react-native';
+import { TouchableWithoutFeedback } from '@kiwicom/margarita-components';
+import { Text, StyleSheet } from '@kiwicom/universal-components';
+import Router from 'next/router';
+import { defaultTokens } from '@kiwicom/orbit-design-tokens';
+
+type Props = {|
+  +label: string,
+  +route: string,
+|};
+
+type State = {|
+  +isActive: boolean,
+|};
+
+export default class NavbarLink extends React.Component<Props, State> {
+  state = {
+    isActive: false,
+  };
+
+  componentDidMount() {
+    if (Router.route === this.props.route) {
+      this.onMouseEnter();
+    }
+  }
+
+  onMouseEnter = () => {
+    this.setState({ isActive: true });
+  };
+
+  onMouseLeave = () => {
+    if (Router.route !== this.props.route) {
+      // If this is the active route, it should always stay marked as active
+      this.setState({ isActive: false });
+    }
+  };
+
+  onPress = () => {
+    Router.push(this.props.route);
+  };
+
+  render() {
+    return (
+      <TouchableWithoutFeedback
+        onMouseEnter={this.onMouseEnter}
```

Comment:
Can we please use `Hoverable` from `universal-components` instead?
```
<Hoverable
   onMouseEnter={this.onMouseEnter}
   onMouseLeave={this.onMouseLeave}
>
```

It disables hover in web version on touch devices. Where `onMouseLeave` is not called and situations like below when component is stuck in hovered mode can happen.

<img width="436" alt="screenshot 2019-01-25 at 10 06 57" src="https://user-images.githubusercontent.com/2660330/51736049-65adad00-2089-11e9-82ed-1a93a6eae3de.png">